server/migrations.js

```diff
@@ -1,26 +1,28 @@
-Migrations.add({
-  version: 1,
-  name: 'Move rounds.progress data to new roundProgresses collection.',
-  up: function() {
-    var allRounds = Rounds.find({}).fetch();
-    allRounds.forEach(function(round) {
-      var done = 0;
-      var total = 0;
-      if(round.progress) {
-        done = round.progress.done;
-        total = round.progress.total;
-      }
-      RoundProgresses.insert({
-        roundId: round._id,
-        competitionId: round.competitionId,
-        done: done,
-        total: total,
+Migrations.add(
```

Comment:
Sweet! Thanks for doing this. As I'm sure you know, until we start getting competitions to use CCM, it's ok to not write db migrations, we can just reset the db. But, this is definitely a good habit to get into!